src/Resources/public/core.js

```diff
@@ -898,7 +898,7 @@ var Backend =
 		var opt = options || {},
 			maxWidth = (window.getSize().x - 20).toInt(),
 			maxHeight = (window.getSize().y - 192).toInt();
-		if (!opt.id) opt.id = 'tl_select';
+		if (!opt.id) opt.id = 'tl_listing';
```

Comment:
You must not change this, because it is not backwards compatible with the old `PageTree` and `FileTree` widgets.